packages/ember-simple-auth/package.json

```diff
@@ -43,7 +43,6 @@
     "ember-cli-content-security-policy": "~1.1.1",
     "ember-cli-dependency-checker": "^3.2.0",
     "ember-cli-htmlbars": "^5.0.0",
-    "ember-cli-htmlbars-inline-precompile": "^3.0.1",
```

Comment:
Is removing this package necessary to upgrade ember-test-selectors? Could we do it separately instead?

Reply:
This is a warning in the console saying to remove it.

Do you plan on doing a minor bump for every PR?  If not, I would like to avoid the cycle of PR, wait time for a review, merge, and then back again.

Reply:
No, I don't plan to release each end every PR separately. It's just better for changelogs to have things separated I think.